test/test_session.py

```diff
@@ -9,6 +9,47 @@
 from radiant_mlhub.session import Session, get_session
 
 
+class TestOverwriteRootURL:
+    def test_default_root_url(self):
+        # Use anonymous session since we don't need to make actual requests
+        session = Session(api_key=None)
+
+        assert session.root_url == Session.DEFAULT_ROOT_URL
+
+    def test_env_variable_root_url(self, monkeypatch):
+        custom_root_url = "https://some-other-url"
+        monkeypatch.setenv('MLHUB_ROOT_URL', custom_root_url)
+        # Use anonymous session since we don't need to make actual requests
+        session = Session(api_key=None)
+
+        assert session.root_url == custom_root_url
+
+        
+
+    @pytest.mark.vcr
+    def test_request_to_custom_url(self, monkeypatch):
+        custom_root_url = "https://www.google.com"
+        monkeypatch.setenv('MLHUB_ROOT_URL', custom_root_url)
+        # Use anonymous session since we don't need to make actual requests
+        session = Session(api_key=None)
+
+        r = session.request("GET", "")
+
+        assert r.request.url == custom_root_url + "/"
+
+
+
+    @pytest.mark.vcr
+    def test_request_to_custom_url(self, monkeypatch):
+        custom_root_url = "https://www.google.com"
+        monkeypatch.setenv('MLHUB_ROOT_URL', custom_root_url)
+        # Use anonymous session since we don't need to make actual requests
+        session = get_session()
```

Comment:
```suggestion
        session = get_session(profile="__anonymous__")
```

I forgot to mention this, but we actually need to pass in a special `profile` value here to make a request without an API key.

Reply:
Actually, it looks like there is a bug in using `profile="__anonymous__"` in `get_session`. I'll fix this in a separate PR, but we can just adjust the URL to include an API key for now.